flanker-test-app/src/main/java/edu/northwestern/mobiletoolbox/flanker/test_app/MainActivity.kt

```diff
@@ -0,0 +1,37 @@
+package edu.northwestern.mobiletoolbox.flanker.test_app
+
+import android.os.Bundle
+import dagger.android.support.DaggerAppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.flanker
+import org.sagebionetworks.research.domain.repository.TaskRepository
+import org.sagebionetworks.research.mobile_ui.perform_task.PerformTaskActivity
+import org.sagebionetworks.research.presentation.model.TaskView
+import java.util.UUID
+import javax.inject.Inject
+
+class MainActivity : DaggerAppCompatActivity() {
+//    MSS-mobile-kit
+
+    @Inject
+    lateinit var taskRepository: TaskRepository
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        flanker.setOnClickListener {
+            launchTask("Flanker", UUID.randomUUID())

```

Comment:
Flanker can be moved to string.xml

Reply:
This is an identifier, not a human readable string. But I can move it